{{ cookiecutter.__package_name_kebab_case }}/.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
For my understanding, what is the benefit of this? As I understand it this will avoid copying the `.git` folder into the Docker context which is available during the build process. But we don't actually copy this directory from the context in to the Docker image, do we? I'd prefer not to add files to the repo root if possible unless there's a clear benefit to doing so.

Reply:
This line will copy everything including `.git`: https://github.com/radix-ai/poetry-cookiecutter/blob/4df178fcaf036ac89387bfde546b23104466385b/%7B%7B%20cookiecutter.__package_name_kebab_case%20%7D%7D/Dockerfile#L144. So I think this is a necessary change.

Reply:
Makes sense.